bedrock/base/templates/404.html

```diff
@@ -13,6 +15,18 @@
   {% stylesheet 'page_not_found' %}
 {% endblock %}
 
+{% block js %}
+  {% if LANG.startswith('en-') and switch('mozorg-search-general') %}
+    {% javascript 'custom-search' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% if LANG.startswith('en-') and switch('mozorg-search-general') %}
+    {{ google_custom_search(settings.GOOG_CUSTOM_SEARCH_TYPES['general'], 'moz-search-general', _('Internet Health'), _('Volunteer, Events, Add-ons, Firefox, Internet Health')) }}
```

Comment:
Hmm... this works, but I was more thinking you'd just pass `'general'` here and expand it in the macro code.